index.js

```diff
@@ -13,8 +13,8 @@
 
 module.exports = {
   parser: 'babel-eslint',
-  extends: 'airbnb',
-  plugins: ['backpack'],
+  extends: ['airbnb', 'plugin:skyscanner-dates/warn'],
+  plugins: ['backpack', 'skyscanner-dates'],
```

Comment:
I'm lazy.... 😇 How come we need both an `extends` & a `plugins` entry here? I know we briefly covered this last week but ... I still don't understand.

Reply:
`plugins` it's like requiring the package, `extends` is actually leveraging one of the built-in configs

Reply:
`plugin:skyscanner-dates/warn` is a shorthand for setting all the 4 rules to `warn` 😉

Reply:
Nice one - thanks.